porch/repository/pkg/git/git.go

```diff
@@ -396,16 +404,27 @@ func (r *gitRepository) discoverFinalizedPackages(main *plumbing.Reference) ([]r
 		return nil, err
 	}
 
+	var revision string
+	rev := ref.Name().String()
+	switch {
+	case strings.HasPrefix(rev, refTagsPrefix):
+		revision = rev[len(refTagsPrefix):]
```

Comment:
FYI, I like strings.TrimPrefix here.  It would be nice if there was a way to avoid having to duplicate HasPrefix TrimPrefix (maybe if TrimPrefix returned a bool as well), but I haven't found one.

Reply:
sg. I'll switch to TrimPrefix. easy way to avoid accidental +-1 error :)